frontend/src/router/index.ts

```diff
@@ -98,6 +98,23 @@ const routes: Array<RouteRecordRaw> = [
           }
         ]
       },
+      {
+        path: 'projects',
+        name: RouteName.HOUSING_PROJECTS,
+        component: () => import('@/views/housing/project/ProjectListView.vue'),
+        meta: {
+          access: [Permissions.NAVIGATION_HOUSING]
+        }
+      },
+      {
+        path: 'project',
+        name: RouteName.HOUSING_PROJECT,
+        component: () => import('@/views/housing/project/ProjectView.vue'),
+        props: createProps,
+        meta: {
+          access: [Permissions.NAVIGATION_HOUSING]
+        }
+      },
```

Comment:
Side note for us to look in to. Can vue-router do pathing like express? Can we have `project` & `project/${id}`? Then we can drop this pluralization we have to do

Reply:
Implemented as in express with path params. No more pluralization!